LICENSE.adoc

```diff
@@ -1,3 +1,5 @@
+[[LICENSE]]
+= LICENSE
```

Comment:
Please use `[#LICENSE]` instead of `[[LICENSE]]`. Same with the other anchors.

Reply:
As far as I understand, Asciidoctor 1.5.3 (that we run tests against) doesn't support such syntax.

Reply:
As @mojavelinux [said in gitter](https://gitter.im/asciidoctor/asciidoctor?at=5e289bcf075a19397ceb60f1):

>> Is it true that old Asciidoctor (at least, 1.5.3) does not support [#anchor] anchor declaration syntax and only supports [[anchor]]?
>
> on top of the document title, yes

So unfortunately, I have to refuse changing this to `[#LICENSE]` because we use this file when testing against Asciidoctor 1.5.3.

Reply:
Yep, that's fine. Perhaps add a line comment to say why, but that's up to you.